.github/workflows/ci.yml

```diff
@@ -12,12 +12,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0', '8.1', '8.2', '8.3']
```

Comment:
Note: 7.4 is already in the `include` section, so does not need to be duplicated here.